pkg/schedule/checker/checker_controller.go

```diff
@@ -233,3 +417,39 @@ func (c *Controller) GetPauseController(name string) (*PauseController, error) {
 		return nil, errs.ErrCheckerNotFound.FastGenByArgs()
 	}
 }
+
+// PatrolRegionContext is used to store the context of patrol regions.
```

Comment:
Do we still need it?

Reply:
it will be used in the next pr, it may be more suitable to move it to the next pr?

Reply:
Should the fields be part of the checker controller? Is it necessary to wrap it?

Reply:
I think it is needed. It will contains more value and config https://github.com/tikv/pd/blob/ae0778f4d80df70f415f8bf60d87b39601fc6471/pkg/schedule/checker/checker_controller.go#L465